remme/ws/events.py

```diff
@@ -150,7 +150,8 @@ def unsubscribe(self, web_sock):
 
         # Validate the response type
         if msg.message_type != Message.CLIENT_EVENTS_SUBSCRIBE_RESPONSE:
-            LOGGER.error(f"Unexpected message type")
+            LOGGER.error(f"Skip unexpected message type")
```

Comment:
I think this should be `DEBUG`, because it doesn't lead to any fails and is not necessary to be seen by an end user.